pkgs/tools/admin/awscli2/default.nix

```diff
@@ -67,6 +67,7 @@ with py.pkgs; buildPythonApplication rec {
   postPatch = ''
     substituteInPlace setup.cfg \
       --replace "colorama>=0.2.5,<0.4.4" "colorama" \
+      --replace "cryptography>=3.3.2,<37.0.0" "cryptography" \
```

Comment:
not sure if this is cause of mass rebuilds, but it would be good to get this to master, as awscli2 is broken on master for now

Reply:
Good point, I've submitted https://github.com/NixOS/nixpkgs/pull/178313 which should be easier to approve.

Reply:
Miss hydra emails when things just stop working :P